test/platform/test-platform-identify.ts

```diff
@@ -59,11 +57,8 @@ test('trigger-identify-with-undefined-verificationId-throws', async t => {
       },
     };
   };
-  app
-    .onIdentify(invalidIdentifyHandler)
-    .onExecute(validExecuteHandler)
-    .listen();
-  const mockLocalHomePlatform = extractMockLocalHomePlatform(app);
+  app.onIdentify(invalidIdentifyHandler).onExecute(executeHandler).listen();
+  const mockLocalHomePlatform = extractStubs(app).mockLocalHomePlatform;
```

Comment:
You can do destructuring `const {mockLocalHomePlatform} = extractStubs(app)`

Reply:
Done